modules/bookAdditon.js

```diff
@@ -0,0 +1,89 @@
+class SavedObjects {
+  constructor() {
+    this.objects = [];
+    this.objectsContainer = document.getElementById('objectsContainer');
+    this.submitButton = document.getElementById('submitBtn');
+    this.titleInput = document.getElementById('title');
+    this.descriptionInput = document.getElementById('description');
+    this.successMessage = document.getElementById('successMessage');
+
+    this.submitButton.addEventListener('click', this.handleSubmit.bind(this));
+  }
+
+  addObject(object) {
+    this.objects.push(object);
+    localStorage.setItem('savedObjects', JSON.stringify(this.objects));
+  }
```

Comment:
- [ ] Please ensure that you use arrow function syntax as opposed to regular function as part of the project requirement. 

A possible solution would be: :arrow_down: 

` addObject = (object) => {`
   ` this.objects.push(object);`
  `  localStorage.setItem('savedObjects', JSON.stringify(this.objects));`
`  }`


- [ ] Please ensure that you replace most of your regular functions with Es6 arrow syntax in all your js files. :slightly_smiling_face: